components/event-bus/internal/knative/util/kn-eventing_test.go

```diff
@@ -163,7 +167,8 @@ func Test_SendMessage(t *testing.T) {
 	assert.Nil(t, err)
 	u, err := url.Parse(srv.URL)
 	assert.Nil(t, err)
-	ch.Status.SetAddress(u.Hostname() + ":" + u.Port())
+	url := &apis.URL{Scheme: "http", Host: fmt.Sprintf("%s:%s", u.Hostname(), u.Port())}
+	ch.Status.SetAddress(url)
```

Comment:
for readability, can we create "url" as a value and pass it's reference in SetAddress()? something like SetAddress(&url)

Reply:
Thanks for the comment.
IMHO, it's readable and I'd not also change the signature of the `SetAddress()`